js/popups/tablemenupopup.ts

```diff
@@ -334,6 +340,12 @@ export class TableMenuPopup {
             leaved.then(removeCurrentTable);
         }
     }
+    private toggleSkipDeals() {
+        this.skipDeals(!this.skipDeals());
+        const currentIndex = tableManager.currentIndex();
+        const tableView = tableManager.tables()[currentIndex];
+        tableView.toggleSkipDeals(this.skipDeals());
```

Comment:
It looks like ActionBlock is already have such functionality and `tableView.actionBlock.skipDeals(this.skipDeals())` do the actual trick. At the same time, the current subscription on the 
https://github.com/online-poker/poker-html-client/blob/3e94a7093f97061cdfe049cb427e56e165614660/js/table/actionBlock.ts#L552
indicate that we have code small in that place. Action block should not know about API calls, so you could hide relationship to API by calling `this.tableView.toggleSkipDeals(value)` there.

That way we could at least move subscription later.